runtime/interpreter/interpreter.go

```diff
@@ -2488,6 +2489,176 @@ var fromStringFunctionValues = func() map[string]fromStringFunctionValue {
 	return values
 }()
 
+type fromBigEndianBytesFunctionValue struct {
+	receiverType sema.Type
+	hostFunction *HostFunctionValue
+}
+
+func padWithZeroes(b []byte, expectedLen int) []byte {
+	l := len(b)
+	if l > expectedLen {
+		panic(errors.NewUnreachableError())
+	} else if l == expectedLen {
+		return b
+	}
+
+	res := make([]byte, expectedLen)
```

Comment:
It might be better to extend the capacity of the existing slice, instead of allocating a whole new slice

Reply:
Done, now we reuse the slice if the capacity >= what we need. Else we have to allocate a new slice.